be/src/util/coding.h

```diff
@@ -51,6 +53,15 @@ inline void encode_fixed64_le(uint8_t* buf, uint64_t val) {
 #endif
 }
 
+inline void encode_fixed128_le(uint8_t* buf, uint128_t val) {
+#if __BYTE_ORDER == __LITTLE_ENDIAN
+        memcpy(buf, &val, sizeof(val));
+    #else
```

Comment:
Indent. Currently, you could refer to https://github.com/apache/incubator-doris/blob/master/docs/documentation/cn/developer-guide/format-code.md to format your code